utils/src/main/java/ai/elimu/analytics/utils/converter/CursorToVideoLearningEventGsonConverter.kt

```diff
@@ -14,16 +14,16 @@ object CursorToVideoLearningEventGsonConverter {
     
     private const val TAG = "CursorToVideoLearningEventGsonConverter"
 
-    // 👇 WARNING: To ensure backward compatibility, these column name values must never be renamed.
-    const val COLUMN_NAME_ID = "column_name_id"
-    const val COLUMN_NAME_ANDROID_ID = "column_name_android_id"
-    const val COLUMN_NAME_PACKAGE_NAME = "column_name_package_name"
-    const val COLUMN_NAME_TIMESTAMP = "column_name_timestamp"
-    const val COLUMN_NAME_ADDITIONAL_DATA = "column_name_additional_data"
-    const val COLUMN_NAME_LEARNING_EVENT_TYPE = "column_name_learning_event_type"
-    const val COLUMN_NAME_VIDEO_TITLE = "column_name_video_title"
-    const val COLUMN_NAME_VIDEO_ID = "column_name_video_id"
-    // ☝️ WARNING
+    // ⚠️ WARNING: To ensure backward compatibility, these bundle key values must never be renamed!
+    const val BUNDLE_KEY_ID = "bundle_key_id"
+    const val BUNDLE_KEY_ANDROID_ID = "bundle_key_android_id"
+    const val BUNDLE_KEY_PACKAGE_NAME = "bundle_key_package_name"
+    const val BUNDLE_KEY_TIMESTAMP = "bundle_key_timestamp"
+    const val BUNDLE_KEY_ADDITIONAL_DATA = "bundle_key_additional_data"
+    const val BUNDLE_KEY_LEARNING_EVENT_TYPE = "bundle_key_learning_event_type"
+    const val BUNDLE_KEY_VIDEO_TITLE = "bundle_key_video_title"
+    const val BUNDLE_KEY_VIDEO_ID = "bundle_key_video_id"
```

Comment:
Due to duplications, I suggest moving those bundle keys from `*Converter` classes to `BundleKeys.kt` .